12_cookies.html

```diff
@@ -38,20 +20,56 @@
               <th>Notes</th>
             </tr>
             <tr>
-              <td>GET</td>
-              <td id='get-cookie'>/session/{sessionId}/cookie</td>
+              <td>POST</td>
+              <td id='post-cookie'>/session/{sessionId}/cookie</td>
               <td></td>
             </tr>
           </table>
-          <p>Remote ends MUST return all cookies that are available in document.cookie via Javascript. In addition, remote ends SHOULD return any HTTP-Only cookies that are associated with the current page.
-        Possible errors that can be returned:
-        <ul>
-          <li><code><a href="#status-invalid-cookie-domain">invalid cookie domain</a></code> - If the cookie's domain is not visible from the current page.<br />
-        </ul>
-        <dl class='parameters'>
-          <dt>optional DOMString? name</dt>
-          <dd>The name (e.g. <code>name=</code>) of the cookie that needs to be returned.</dd>
-        </dl>
+          <p>The <a>remote end steps</a> for the <a>getCookie</a> command are:</p>
+          <ol>
+            <li><p>If the <a>current browsing context</a> is <a>no longer open</a>, return an error with code <a>no such window</a>.</p></li>
+            <li><p>Let <var>name</var> be the result of <a>getting a property</a> named "name" from the <var>parameters</var> argument.</p></li>
+            <li><p>Let <var>result</var> be an empty JSON List</p></li>
+            <li><p>Let <var>cookies</var> be a list of all cookies [[!RFC6265]] in the cookie store associated with <a href="https://dom.spec.whatwg.org/#concept-document-url"> the current document or frame address</a></p></li>
+            <li><p>Let <var>length</var> be the length of <var>cookies</var>.</p></li>
+            <li><p>Let <var>k</var> be 0</p></li>
+            <li><p>While <var>k</var> < <var>length</var>: </p>
+              <ol>
+                <li><p>Let <var>cookie</var> be the value in <var>cookies</var> at index <var>k</var>.</p></li>
+                <li><p>If <var>name</var> is undefined:</p>
+                  <ol>
+                    <li><p>Append a <a>serialised cookie</a> to <var>result</var>.</p></li>
+                  </ol>
+                </li>
+                <li><p>if <var>name</var> is defined and is equal to <var>cookie-name</var>:</p>
```

Comment:
in [rfc6265](http://tools.ietf.org/html/rfc6265#section-4.1.1) in the `cookie-pair` is `cookie-name = cookie-value` so would like to preserve some of the semantics by using `cookie-name`